models/news.models.js

```diff
@@ -25,3 +25,16 @@ exports.selectArticleCommentsById = (article_id) => {
 		return comments.rows
 	})
 }
+
+exports.updateArticleVotes = (article_id, body) => {
+	const values = [article_id, body]
+	const pSelectArticle = this.selectArticle(article_id)
+	const pUpdate = db.query('UPDATE articles SET votes = votes + $2 WHERE article_id = $1 RETURNING *;', values)
```

Comment:
Naming wise - it would be confusing for anyone else to take over. Im not sure what the random p bit is at the start of the names, presume PSQL?
Readability should always be number one priority, especially thinking about others working on/maintaining the same code.

Reply:
Was meant to be indicative of an instance of a promise, but i get you and will change :)